aloscene/depth.py

```diff
@@ -132,7 +138,11 @@ def encode_absolute(self, scale=1, shift=0, prior_clamp_min=None, prior_clamp_ma
         if prior_clamp_min is not None or prior_clamp_max is not None:
             depth = torch.clamp(depth, min=prior_clamp_min, max=prior_clamp_max)
 
-        depth[torch.unsqueeze(depth < 1e-8, dim=0)] = 1e-8
+        if not keep_negative:
```

Comment:
If depth is_planar False then the encode_absolute should not clip value bellow zero, otherwise if is_planar: the negative values are clip. Thus, keep_negative might not be necessary. It should be automatic based of self.is_planar